doc: known_issues: Add NCSDK-13923 (#6954)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

  **Workaround:** Change ``pwm_pin_set_cycles`` to ``pwm_pin_set_usec`` in function :c:func:`led_pwm_set_brightness` in Zephyr's driver :file:`led_pwm.c` file.

nRF Machine Learning
====================

.. rst-class:: v1-9-0

> NCSDK-13923: Device may crash when Bluetooth Central establishes security

Reviewer: ```suggestion
NCSDK-13923: Device may crash when Bluetooth Central establishes secured connection
```
Would this work? "Established security" sounds like slang. Also, why "may"? What are the conditions on which this "may" depends? Can the stack size be large enough for some supported devices? Can it increase with some Kconfig options?

Author: The secure connections in Bluetooth refers to option for pairing (LE Secure Connections). I will rephrase it a bit.

Author: I think the stack may overflow, but it seems like there are also cases when connection security is properly established. It depends on operations that are actually performed over Bluetooth and implementation, I would not go into details here .

>   The crash is caused by too small Bluetooth RX thread stack size.

Reviewer: ```suggestion
  The device programmed with the nRF Machine Learning application may crash when Bluetooth Central establishes a secured connection because of insufficient Bluetooth RX thread stack size.
```

>   The device programmed with the nRF Machine Learning application may crash during Bluetooth bonding  because of insufficient Bluetooth RX thread stack size.

Reviewer: ```suggestion
  The device programmed with the nRF Machine Learning application may crash during Bluetooth bonding because of insufficient Bluetooth RX thread stack size.
```
Just a nit...

Author: Thanks

>   **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``4870fcd8316bd3a4b53ca0054f0ce35e1a8c567d``).

Reviewer: ```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``4870fcd8316bd3a4b53ca0054f0ce35e1a8c567d``).
```
